### DISCUSSION TITLE
NVIDIA graphics cards for spark programming

### DISCUSSION THREAD

**Discussion by jschwar313** [NVIDIA/spark-rapids#6068] [Q&A] [answered]
I've tried to find out which NVIDIA graphics cards are eligible for spark programming and I can't find that.  Where can I find such a list?

**Comment by abellina** [comment, marked as answer]
Hello @jschwar313. We have an FAQ entry about this here: https://nvidia.github.io/spark-rapids/docs/FAQ.html#what-hardware-is-supported. 

Please let us know if this doesn’t answer your question!

**Reply by benbowles** [reply]
Hi @abellina  , I am using a GeForce 1070. Will it work? It says Volta or better below. 1070 is Pascal right? I got everything setup, then I see this error: "java.lang.IllegalStateException: Could not find a single GPU to use"
I would like to confirm this is because of the incompatbility GPU card if thats OK.
Thanks in advance!!



The plugin is tested and supported on P100, V100, T4, A2, A10, A30 and A100 datacenter GPUs. It is possible to run the plugin on GeForce desktop hardware with Volta or better architectures. GeForce hardware does not support [CUDA forward compatibility](https://docs.nvidia.com/deploy/cuda-compatibility/index.html#forward-compatibility-title), and will need CUDA 11.5 installed. If not, the following error will be displayed:

**Reply by jlowe** [reply]
Pascal is supported as long as forward compatibility is not required in the environment.  So if you have CUDA 11.5 installed in the environment, Pascal GPUs should work.  If you're still having issues, it would be good to post `nvidia-smi` output in the environment being used for the executors.  Also the RAPIDS Accelerator will log INFO-level messages on the executor for GPUs it sees but cannot use (message will start with, "Will not use GPU").
